engineering/principles-and-practices.md

```diff
@@ -12,20 +12,20 @@ meeting with the group and everyone says 'yea' to a change, it can simply be mer
 
 As an engineering team, we seek to:
 
-- Build an esprit de corps where the engineering team is inspired to ship high quality software on a predictable schedule
-- Create a culture of trust where engineers feel like they have the freedom to do great work, share vulnerabilities, and inquire
-- Create a culture of inclusion where all engineers feel like they have voice
-- Create a culture of ownership where engineers take responsibility for shipping high quality code, system performance and uptime, and client happiness
-- Foster an environment where engineers are challenged by the work and they strive for excellence and continued improvement
+- Build an [esprit de corps](https://www.dictionary.com/browse/esprit-de-corps) where the engineering team is inspired to ship high quality software on a predictable schedule
+- Create a culture of trust where engineers feel they have the freedom to do great work, share vulnerabilities, and inquire
+- Create a culture of inclusion where all engineers feel they have a voice
+- Create a culture of ownership where engineers take responsibility for shipping high quality code, system performance & uptime, and client happiness
+- Foster an environment where engineers are challenged, strive for excellence and continually improve
 - Encourage a culture of self-organizing teams
 
 ## Our Mission
-__Our mission is not just writing software.__
+__Our mission is more than writing software.__
```

Comment:
This one, I'm not totally convinced is a good change. "not just."

I read a blog post once about people using the word "just" when giving advice or instruction. Some simple examples:
"Just drop node_modules and run npm install"
"Just rebuild the clustered index"
"Just ..."

It gave me an aversion to the word "just" (in the context of "just do this", not in the context of what is "right & just")

Reply:
I agree with this, it a more positive framing